enhancements/two-node-fencing/tnf.md

```diff
@@ -339,109 +356,138 @@ sshKey: ''
-It relies on resource agents to understand how to check the state of a resource, as well as start and stop them to achieve the desired target state.
-How a given agent uses these actions, and associated states, to model the resource is opaque to the cluster and depends on the needs of the underlying resource.
+RHEL-HA has no real understanding of the resources (IP addresses, file systems, databases, even virtual machines) it manages. It relies on resource agents to understand how to check the state of a
+resource, as well as start and stop them to achieve the desired target state. How a given agent uses these actions, and associated states, to model the resource is opaque to the cluster and depends on
+the needs of the underlying resource.
 
-Resource agents must conform to one of a variety of standards, including systemd, SYS-V, and OCF.
-The latter is the most powerful, adding the concept of promotion, and demotion.
-More information on creating OCF agents can be found in the upstream [developer guide](https://github.com/ClusterLabs/resource-agents/blob/main/doc/dev-guides/ra-dev-guide.asc).
+Resource agents must conform to one of a variety of standards, including systemd, SYS-V, and OCF. The latter is the most powerful, adding the concept of promotion, and demotion. More information on
+creating OCF agents can be found in the upstream [developer guide](https://github.com/ClusterLabs/resource-agents/blob/main/doc/dev-guides/ra-dev-guide.asc).
 
 Tools for extracting support information (must-gather tarballs) will be updated to gather relevant logs for triaging issues.
 
-As part of the fencing setup, the cri-o and kubelet services will still be owned by systemd when running under pacemaker. The main difference is that the resource agent will be responsible for signaling systemd to change their active states.
-The etcd containers are different in this respect since they will be restarted using Podman, but this will be running as root, as it was under CEO.
+As part of the fencing setup, the cri-o and kubelet services will still be owned by systemd when running under pacemaker. The main difference is that the resource agent will be responsible for
+signaling systemd to change their active states. The etcd containers are different in this respect since they will be restarted using Podman, but this will be running as root, as it was under CEO.
+
+#### The Fencing Network
+The [goals](#goals) section of this proposal highlights the intent of minimizing the scenarios that require manual administrator intervention and maximizing the cluster's similarity to the experience
+of running a 3-node hyperconverged cluster - especially in regard to auto-recovering from the failure of either node. The key to delivering on these goals is fencing.
+
+In order be able to leverage fencing, two conditions must be met:
+1. The fencing network must be available
```

Comment:
I'm wondering if we need more details about how to add a fencing network, or if network people know what to do?

Reply:
We do - but I expect those details to be included via @mshitrit's work on https://issues.redhat.com/browse/OCPEDGE-1461

Reply:
This commit just focused on rewrapping the comments that were already there.

Reply:
ok